### ISSUE TITLE
Scrub slider is hard to tap and grab

### ISSUE THREAD

**Issue by TheFlatworm** [BLeeEZ/amperfy#68]
The slider to scrub through a song is very small and difficult to catch and move with a tap - is there any way to make the location indicator bigger so it's easier to scrub?

**Comment by BLeeEZ** [comment]
You are right. I have noticed this difficulty for long songs/podcasts too. I will take at this improvement.

**Comment by TheFlatworm** [comment]
Thank you!!

**Comment by BLeeEZ** [comment]
Implemented in 2d283abba078cb2e3693e0eb005316865e1aa053.
